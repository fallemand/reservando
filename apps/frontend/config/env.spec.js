describe("environmental variables", () => {
  const ENV = {
    BUILD: "false",
    NODE_ENV: "production",
    DATADOG_TRACE_AGENT_HOSTNAME: "DATAGOD-HOSTNAME",
    DATADOG_TRACE_AGENT_PORT: "DATAGOD-PORT",
    DATADOG_STATSD_SERVICE_HOST: "DATAGOD-STATS-HOST",
    DATADOG_STATSD_SERVICE_PORT: "DATAGOD-STATS-POST",
    FIREBASE_PROJECT_ID: "__PROJECT_ID__",
    FIREBASE_DATABASE_URL: "__DATABASE_URL__",
    GOOGLE_APPLICATION_CREDENTIALS: '{"__KEY__": "__VALUE__"}',
  };

  beforeEach(() => {
    jest.resetModules();
    process.env = { ...ENV };
  });

  test("should generate map of values", () => {
    // eslint-disable-next-line global-require
    const env = require("./env");
    expect(env).toMatchSnapshot();
  });
});
