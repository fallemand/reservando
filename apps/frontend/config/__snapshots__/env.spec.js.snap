// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`environmental variables should generate map of values 1`] = `
Object {
  "build": "false",
  "datadogStatsd": Object {
    "host": "DATAGOD-STATS-HOST",
    "port": "DATAGOD-STATS-POST",
  },
  "datadogTrace": Object {
    "host": "DATAGOD-HOSTNAME",
    "port": "DATAGOD-PORT",
  },
  "env": "production",
  "isDevelopment": false,
  "isProduction": true,
  "isTesting": false,
  "port": 3000,
}
`;
